src/components/ConcernList/ConcernList.tsx

```diff
@@ -8,13 +12,213 @@ interface ConcernListProps {
 }
 
 function ConcernList({ concernList }: ConcernListProps) {
+  const [currentCtegory, setCurrentCtegory] = useState(Category.Love)
+  const [currentDistance, setCurrentDistance] = useState(Distance.D3)
+  const [currentEmotion, setCurrentEmotion] = useState(Emotion.Happy)
```

Comment:
고민리스트의 정보를 들고 있는 state  들이기때문에
취향차이겠지만 저는 이런경우  오브젝트로 관리하는걸 선호합니다!
대용쓰의 의견도 궁금하네요! 

```suggestion
  const [currentInfo, setCurrentInfo] = useState({love:Category.Love, d3: Distance.D3, happy: Emotion.Happy})
```

Reply:
정신이 없어서 한참전에 했는데 이제 확인했네요 음 좀더 생각했으면 더 좋은구조로 짰겠지만...시간이 없다보니 손가는데로 짰네요ㅋㅋㅋㅋㅋㅋ